src/launcher/components/AppManagementFilter.jsx

```diff
@@ -26,6 +26,25 @@ export const sortedSources = sources => {
     return [...officialsAndLocal, ...rest];
 };
 
+function addSearchFieldShortcuts() {
+    const searchField = document.querySelector('.filterbox input');
+
+    if (searchField) {
+        document.addEventListener(
+            'keydown',
+            event => {
+                const keyName = event.key;
+                if (event.ctrlKey) {
+                    if (keyName === 'e' || keyName === 'f' || keyName === 'l') {
```

Comment:
Very minor suggested change:

```suggestion
                const { key } = event;
                if (event.ctrlKey) {
                    if (['e', 'f', 'l'].includes(key)) {
```